frontend/src/scenes/notebooks/NotebooksScene.tsx

```diff
@@ -24,9 +29,47 @@ export function NotebooksScene(): JSX.Element {
                     </div>
                 }
                 buttons={
-                    <LemonButton data-attr={'new-notebook'} to={urls.notebook('new')} type="primary">
-                        New notebook
-                    </LemonButton>
+                    <>
+                        <FlaggedFeature flag="posthog-3000" match="test">
+                            <LemonMenu
+                                items={[
+                                    {
+                                        label: 'Load from JSON',
+                                        onClick: () => {
+                                            void selectFiles({
+                                                contentType: 'application/json',
+                                                multiple: false,
+                                            })
+                                                .then((files) => getTextFromFile(files[0]))
+                                                .then((text) => {
+                                                    const data = JSON.parse(text)
+                                                    if (data.type !== 'doc') {
+                                                        throw new Error('Not a notebook')
+                                                    }
+
+                                                    // Looks like a notebook
+                                                    router.actions.push(
+                                                        urls.canvas(),
+                                                        {},
+                                                        {
+                                                            '🦔': base64Encode(text),
```

Comment:
I assume we use Max as a URL param in Canvas?

Reply:
Yep :D Mostly because toolbar uses `#state` so the hedgehog felt safest (and most fun)